.github/workflows/docker-images.yml

```diff
@@ -36,11 +41,33 @@ jobs:
       REPO_URL: ${{ github.event.pull_request.head.repo.html_url }}
     steps:
       - name: Print build information
-        run: 'echo head_ref: "$GITHUB_HEAD_REF", ref: "$GITHUB_REF"'
+        run: 'echo head_ref: "$GITHUB_HEAD_REF", ref: "$GITHUB_REF", triggering sha: ${{ github.sha }}'
 
-      - uses: actions/checkout@v4
+      - name: Checkout omes repo
+        uses: actions/checkout@v4
         with:
+          repository: ${{ inputs.omes-repo-path }}
+          ref: ${{ inputs.omes-repo-ref }}
           submodules: true
+          fetch-depth: 0
+
+      - name: Checkout lang repo
+        if: ${{ inputs.sdk-repo-ref }}
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ inputs.sdk-repo-url }}
+          submodules: recursive
+          path: checked-out-sdk/
+          ref: ${{ inputs.sdk-repo-ref }}
+          fetch-depth: 0
+
+      - name: Set SDK Version Variable
+        run: |
+          if [ -z "${{ inputs.sdk-version }}" ]; then
+              echo "SDK_VERSION=checked-out-sdk/" >> $GITHUB_ENV
+          else
+              echo "SDK_VERSION=${{ inputs.sdk-version }}" >> $GITHUB_ENV
+          fi
```

Comment:
Up to you but this might be less surprising to read if we did the equivalent of

```
--version ${{ env.SDK_VERSION || checked-out-sdk/ }}
```

below (or whatever the correct syntax is for doing that) and didn't set the env var to the strange value.

Reply:
Sure, done. Can't remember why I thought I needed this lol.